src/main/kotlin/ai/digital/integration/server/deploy/tasks/server/DownloadAndExtractDbUnitDataDistTask.kt

```diff
@@ -15,12 +16,16 @@ open class DownloadAndExtractDbUnitDataDistTask : Copy() {
         this.group = PLUGIN_GROUP
         val version = DeployExtensionUtil.getExtension(project).xldIsDataVersion
         if (version != null) {
-            project.buildscript.dependencies.add(
+            project.dependencies.add(
                 SERVER_DATA_DIST,
                 "com.xebialabs.deployit.plugins:xld-is-data:${version}:repository@zip"
             )
-            this.from(project.zipTree(project.buildscript.configurations.getByName(SERVER_DATA_DIST).singleFile))
-            this.into(IntegrationServerUtil.getDist(project))
+            val taskName = "${NAME}Exec"
+            this.dependsOn(project.tasks.register(taskName, Copy::class.java) {
```

Comment:
if task name is changed we need to updated its usage in other repository

Reply:
It is only used inside the project, checking its usecase elsewhere